my_list.rb

```diff
@@ -0,0 +1,26 @@
+require_relative 'enumerable'
+
+class MyList
+  include Enumerable
+  def initialize(*list)
+    @list = list
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
```

Comment:
- [ ] It will be great to ensure that when you create the `each` method, it does not implement its function using each method defined in ruby. The assignment was to create your own `each `method. So kindly create the behaviour of the each method without using the one defined in ruby example ⬇️ 

def each
    return to_enum(:each) unless block_given?

    counter = 0
    while counter < @list.length
      yield(@list[counter])
      counter += 1
    end
    @list
end

Reply:
Make changes as suggested.